### DISCUSSION TITLE
[CS101]請問5-3：網站怎麼儲存資料的－－資料庫簡介 的ＳＱＬ去哪裡下載？

### DISCUSSION THREAD

**Discussion by fediland1478** [Lidemy/forum#287] [後端相關問題]
請問作者演示的ＳＱＬ程式是去哪裡下載？

謝謝！

**Comment by aszx87410** [comment]
影片中用的是 https://eggerapps.at/postico/

可以直接找一些線上平台練習會比較方便，例如說：

1. https://www.db-fiddle.com/
2. http://sqlfiddle.com/

**Reply by fediland1478** [reply]
謝謝你！

因為剛剛還沒有發問前,自己下載了載包,但裡頭還有許多小載包,不知道是下載哪一個才正確！
有線上平台練習,方便許多了！

**Comment by fediland1478** [comment]
<img width="1424" alt="Screenshot 2022-10-17 at 10 53 26 AM" src="https://user-images.githubusercontent.com/100910568/196079428-08669c65-260c-479c-973b-63fee5844b9e.png">


請問：

當我刪除 商品編號 1 後,
顯示結果只有Record count:0
並非Record count:1

而我想用觀看所有欄位來確認是否刪除我所要的欄位時,
卻沒有顯示欄位出來！

一樣出現圖片下方的文字
（Record count:0 ........)

有什麼方法能讓我成功刪除？

謝謝！

**Reply by cwc329** [reply]
你其實已經刪除成功了。
因為你塞的所有資料`產品編號`都是`1`，
所以下 `DELETE FROM test WHERE 產品編號=1;` 就會把所有的資料都刪掉。

**Reply by fediland1478** [reply]
<img width="1430" alt="delete 2" src="https://user-images.githubusercontent.com/100910568/196321603-c722e203-eec2-4d7e-9382-0f00d458e3d6.png">
<img width="1422" alt="delete 3" src="https://user-images.githubusercontent.com/100910568/196321637-64df35ed-f234-4eae-b1d1-ff6e826682fc.png">


謝謝你

**Reply by cwc329** [reply]
BTW
資料庫慣例的欄位命名會用英文，
使用其他語言可能會有編碼以及其他問題要處理。
還有 build schema 慣例上通常只有指建立一個新的 Table 以及相關的基礎架構，
最多會再塞入一些初始資料，
通常不會有 DELETE 的動作。
DELETE 還有 UPDATE 等你想執行的操作最好是在右邊的 SQL 部分去執行，
左邊就放環境建立相關的即可。

**Reply by fediland1478** [reply]
了解,謝謝你讓我知道這些！
